tensorly/contrib/decomposition/tests/test_mps_decomposition_cross.py

```diff
@@ -9,14 +9,11 @@
 from ....random import check_random_state
 from tensorly.testing import assert_
 
-skip_if_tensorflow = pytest.mark.skipif(tl.get_backend() == "tensorflow",
-                                        reason="Operation not supported in TensorFlow")
-skip_if_jax = pytest.mark.skipif(tl.get_backend() == "jax",
-                                        reason="Operation not supported in JAX")
+skip_if_backend = pytest.mark.skipif(tl.get_backend() in ("tensorflow", "jax", "mxnet"),
+                                     reason=f"Operation not supported in {tl.get_backend()}")
```

Comment:
Why are we skipping for MXNet? I am planning to fix this for JAX and tensorflow too, should be mostly a matter of doing the index assignment with index_update.

Reply:
Fixed (mostly). See below.